examples/material_gallery/test/smoke_test.dart

```diff
@@ -7,51 +7,22 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:test/test.dart';
 
 import '../lib/main.dart' as material_gallery;
+import '../lib/gallery/app.dart' as material_gallery;
+import '../lib/gallery/item.dart' as material_gallery;
 
 // Warning: the following strings must be kept in sync with GalleryHome.
 const List<String> demoCategories = const <String>['Demos', 'Components', 'Style'];
-const List<String> demoNames = const <String>[
-  'Weather',
-  'Fitness',
-  'Fancy lines',
-  'Flexible space toolbar',
-  'Floating action button',
-  'Buttons',
-  'Cards',
-  'Chips',
-  'Date picker',
-  'Data tables',
-  'Dialog',
-  'Drop-down button',
-  'Expand/collapse list control',
-  'Grid',
-  'Icons',
-  'Leave-behind list items',
-  'List',
-  'Menus',
-  'Modal bottom sheet',
-  'Over-scroll',
-  'Page selector',
-  'Persistent bottom sheet',
-  'Progress indicators',
-  'Scrollable tabs',
-  'Selection controls',
-  'Sliders',
-  'Snackbar',
-  'Tabs',
-  'Text fields',
-  'Time picker',
-  'Tooltips',
-  'Colors',
-  'Typography'
-];
+
+Finder byRouteName(WidgetTester tester, String routeName) {
```

Comment:
it's not clear what kind of widget this finds. I initially thought it found the ModalRoute.